pkg/operator/encryption/encryptionconfig/config.go

```diff
@@ -153,6 +128,8 @@ func getGRsActualKeys(encryptionConfig *apiserverconfigv1.EncryptionConfiguratio
 func secretsToProviders(desired state.GroupResourceState) []apiserverconfigv1.ProviderConfiguration {
```

Comment:
nit: should we update the function name as well? (we are not dealing with secrets anymore)